examples/6-base-service/src/user.resolver.ts

```diff
@@ -0,0 +1,59 @@
+import { GraphQLResolveInfo } from 'graphql';
+import { Arg, Args, Ctx, Mutation, Query, Resolver } from 'type-graphql';
+import { Repository } from 'typeorm';
+import { InjectRepository } from 'typeorm-typedi-extensions';
+
+import { BaseContext, StandardDeleteResponse, WarthogService } from '../../../src';
+import {
+  UserCreateInput,
+  UserUpdateArgs,
+  UserWhereArgs,
+  UserWhereInput,
+  UserWhereUniqueInput
+} from '../generated';
+
+import { User } from './user.model';
+
+@Resolver(User)
+export class UserResolver {
+  service: WarthogService<User>;
+
+  constructor(@InjectRepository(User) private readonly userRepository: Repository<User>) {
+    this.service = new WarthogService<User>(User, userRepository);
```

Comment:
Instead, we spin up a service that will make all of the generic calls.

Reply:
great! this will make things much more flexible/cleaner when we need flexibility around the DAL